pages/success.tsx

```diff
@@ -1,10 +1,28 @@
-import { Card, Heading, SmallText } from "@cheapreats/react-ui";
+import { Card, Heading, SmallText, Button } from "@cheapreats/react-ui";
 import React from "react";
 import styled from "styled-components";
 import Image from "next/image";
 import Snowfall from "react-snowfall";
+import { useState, useEffect } from "react";
+import { useRouter } from "next/router";
 
 export const Success: React.VFC = () => {
+  const [orderId, setOrderId] = useState<string>("");
+  const router = useRouter();
```

Comment:
Is there no typing needed here like <url: string>
Might reduce the need for AS below